modules/grafana/templates/dashboards/whitehall_health.json.erb

```diff
@@ -445,7 +445,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "target": "aliasSub(groupByNode({elasticsearch,support}-?_backend*.cpu-*.cpu-user,0,\"sumSeries\"),\"_backend\",\"\")",
+              "target": "aliasSub(groupByNode({elasticsearch,support}-*.cpu-*.cpu-user,0,\"sumSeries\"),\"_backend\",\"\")",
```

Comment:
Are these both names of nodes? If so, then then the only Elasticsearch nodes we have are called `rummager-elasticsearch`, and I'm not sure what `support` is.

Reply:
Nice catch, I updated